lib/MusicBrainz/Server/Controller/OtherLookup.pm

```diff
@@ -6,6 +6,26 @@ use Moose::Util qw( find_meta );
 use MusicBrainz::Server::Translation qw( l );
 use MusicBrainz::Server::Constants qw( entities_with );
 
+sub quoted_query {
+    my ($self, $label, $value) = @_;
+
+    # We want to quote the query so Lucene understands it properly,
+    # except if it is a regular expression or the user already entered
+    # a quoted value.
+    my $is_regex = $value =~ /^\/.*\/$/;
+    my $is_quoted = $value =~ /^".*"$/;
+
+    # We escape any quotes inside the actual string so that they are
+    # still searched for when we put the whole thing in quotes.
+    my $escaped_value = $value =~ s/"/\\"/gr;
```

Comment:
You'd only want to do this unless `$is_quoted` though, right? Otherwise you'd escape the outer quotes there.